backend/controllers/userController.js

```diff
@@ -93,10 +93,10 @@ const updateUser = async (req, res) => {
       id: userId,
       lang: language,
     };
-    user.token = jwt.sign(userForToken, process.env.SECRET);
+    res.json({ token: jwt.sign(userForToken, process.env.SECRET) });
```

Comment:
add `return res.json.....` on line 96 otherwise it will throw an error on the backend something like can not send headers twice

Reply:
Ah, sure! Thanks!